### PR TITLE
Make EQ work with IE/edge

### PR DESCRIPTION
**Pull request by jacobjuul**
IE and Edge parses cssRules with [attributes] differently

### CODE REVIEW

File: src/ElementQueries.js [hunk 1]
```diff
@@ -187,10 +187,12 @@
         function extractQuery(css) {
             var match;
             var smatch;
+            var ieSelector = '.' + css.split('.')[1].trim();
```
Comment (marcj) [inline src/ElementQueries.js]: what if the rule has no `.` in it?

**Comment by marcj** [comment]
Hey cool, thanks! so comma separated rules are OK in IE/Edge now?

**Comment by jacobjuul** [comment]
@marcj No they need to be specified individually. But at least it works now. ;)

File: src/ElementQueries.js (hunk 1 above)
Comment (jacobjuul) [inline src/ElementQueries.js]: good point. I haven't tested how IE handles ID's.

Comment (jacobjuul) [inline src/ElementQueries.js]: @marcj fixed in new commit

**Comment by marcj** [comment]
Heyho! We've pulled in some code lately which should fix this already. @jacobjuul, can you please confirm that your issue is fixed with current master version?

**Pull request closed without merging**